app/src/main/java/com/beemdevelopment/aegis/otp/GoogleAuthInfo.java

```diff
@@ -180,6 +193,13 @@ public static GoogleAuthInfo parseUri(Uri uri) throws GoogleAuthInfoException {
         return Base32.decode(s);
     }
 
+    /**
+     * Decodes first 26 bytes, cause next bytes means UID, Pin length and checksum - not necessary
```

Comment:
Why do we not need to know the PIN length and checksum?

Reply:
Here is my mistake.
From QR the secret comes clean, with only 26 characters inside.
But when you enter it manually, the Yandex site gives out a secret 43 characters long, with a checksum at the end.

I added validation for the second case to make it easier for the user to check if he copied/typed his secret correctly.